cypress-io/cypress
# misc: Update styling of Runs Page

- Closes #26180 

### Additional details
* Updating status icons to match other areas of the app (specs list, debug page)
  * Use shared component to reduce complexity and use icon assets from design system
* Wrap run status as viewport shrinks to improve responsive layout
* Add link params

### Steps to test
Review `RunCard.cy.tsx` component test. Changes can be viewed in the following tests:

- "renders expected icon for each run status"
- "renders with all run information on small viewport"

### How has the user experience changed?

![Screenshot 2023-04-03 at 10 09 41 AM](https://user-images.githubusercontent.com/11482842/229560018-75f29ebd-f223-42d2-95f4-bb866cca061d.png)

![Screenshot 2023-04-03 at 10 29 56 AM](https://user-images.githubusercontent.com/11482842/229560033-9a49365f-ef8c-4ba9-96f4-1aa1dcc6cc85.png)

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
